linkerd/proxy/api-resolve/src/resolve.rs

```diff
@@ -20,7 +20,6 @@ use tracing::{debug, info, trace};
 #[derive(Clone)]
 pub struct Resolve<S> {
     service: DestinationClient<S>,
-    scheme: String,
```

Comment:
does the control plane mind that we're no longer sending schemes?

Reply:
We've been sending empty strings for a long time

Reply:
oh, i see, this just removes it from the struct. cool, carry on!